blocks/filter.py

```diff
@@ -96,6 +96,16 @@ class object contained in the list.
     """
     def __init__(self, roles=None, bricks=None, each_role=False, name=None,
                  name_regex=None, applications=None):
+        if bricks is not None and not isinstance(bricks, Brick):
```

Comment:
You are right, but the input control at https://github.com/bartvm/blocks/pull/578/files#diff-b78ec68afdd58167518263ff5b423d3aR97 wouldn't allow a non-class object.

I guess `isclass` can be avoided here, while `issubclass(brick, Brick)` should be added as input check.